hoot-core/src/main/cpp/hoot/core/util/OsmUtils.cpp

```diff
@@ -138,4 +144,97 @@ QString OsmUtils::currentTimeAsString()
   return QDateTime::currentDateTime().toString("yyyy-MM-ddThh:mm:ssZ");
 }
 
```

Comment:
These methods can be cleaned up with abstraction, just don't feel like doing it right now...later.